h/views/api_auth.py

```diff
@@ -20,6 +20,7 @@ def access_token(request):
         'access_token': token.value,
         'token_type': 'bearer',
         'expires_in': TOKEN_TTL.total_seconds(),
+        'refresh_token': token.refresh_token,
```

Comment:
Ok, I've pushed this change.

I noticed that the view code always returns an `expires_in` even if the token does not have an expiry time. It assumes that the oauth service always returns tokens that expire. Sounds like it would be worth making this also check whether or not the token has an expiry time? (Although then we'd have to change how `expires_in` is calculated).

<pre><code>
response = {
    'access_token': token.value,
    'token_type': 'bearer',
    <strong>'expires_in': TOKEN_TTL.total_seconds()</strong>,
}
</code></pre>